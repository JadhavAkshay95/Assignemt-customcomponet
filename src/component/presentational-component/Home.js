import React from "react";
import EmpFilterContainer from "../container-component/EmpFilterContainer";
import EmpListContainer from "../container-component/EmpListContainer";
import Header from "./Header";

const Home = () => {
  return (
    <>
      <Header />
      <div className="emp-container">
        <EmpFilterContainer />
        <div className="emp-list-container">
          <EmpListContainer />
        </div>
      </div>
    </>
  );
};

export default Home;
