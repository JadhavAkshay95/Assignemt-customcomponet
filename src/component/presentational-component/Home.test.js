import { shallow } from "enzyme";
import Home from "./Home";
import React from "react";

describe("Home Component", () => {
  it("Should render Home component", () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});
