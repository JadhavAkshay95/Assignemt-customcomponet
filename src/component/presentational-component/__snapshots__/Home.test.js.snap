// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component Should render Home component 1`] = `
<Fragment>
  <Header />
  <div
    className="emp-container"
  >
    <Connect(EmpFilter) />
    <div
      className="emp-list-container"
    >
      <Connect(EmployeeList) />
    </div>
  </div>
</Fragment>
`;
